src/Extensions/Banshee.YouTube/Banshee.YouTube.csproj

```diff
@@ -41,19 +41,14 @@
     <Compile Include="Banshee.YouTube.Gui\VideoStreamTile.cs" />
   </ItemGroup>
   <ItemGroup>
+    <PackageReference Include="Google.GData.Client" Version="2.2.0.0" />
+    <PackageReference Include="Google.GData.Extensions" Version="2.2.0.0" />
+    <PackageReference Include="Google.GData.YouTube" Version="2.2.0.0" />
+    <PackageReference Include="Newtonsoft.Json" Version="4.0.5.0" />
```

Comment:
what's pulling in this dependency?

Reply:
All the used ```Google.GData.*``` packages.

See https://www.nuget.org/packages/Google.GData.Client/ for example.